source/documentation/tdb/tdb-xloader.md

```diff
@@ -3,20 +3,24 @@ title: TDB xloader
 ---
 
 TDB xloader ("x" for external) is a bulkloader for very large datasets. The goal
-is stability and reliability for long running loading, running on modest and
+is stability and reliability for long running loading, running on modest
+hardware and can load to storage of rotating disk or SSD.
```

Comment:
I didn't understand the "can load to storage of rotating disk", but might be just lack of coffee :coffee: :laughing:

Reply:
Or too much coffee (in my timezone)!